source/language/types.rst

```diff
@@ -368,54 +510,112 @@ their argument expression in parentheses:
       +----------+-----------------------------------+--------------------------------------+----------------------------------------+
-      | pow      | ``int``, ``uint``                 | ``int``                              |                                        |
+      | pow      | ``int``, ``uint``                 | ``int``                              | :math:`\texttt{pow(a, b)} = a^b`.      |
       |          |                                   |                                      |                                        |
       |          | ``float``, ``float``              | ``float``                            | For floating-point and complex values, |
       |          |                                   |                                      | the principal value is returned.       |
       |          | ``complex``, ``complex``          | ``complex``                          |                                        |
       +----------+-----------------------------------+--------------------------------------+----------------------------------------+
-      | rotl     | ``bit[n]``, (``int`` or ``uint``) | ``bit[n]``                           |                                        |
+      | rotl     | ``bit[n]``, ``int``               | ``bit[n]``                           | Rotate the bits in the representation  |
+      |          |                                   |                                      | ``n`` to the left (towards higher      |
+      |          |                                   |                                      | indices).  This is similar to a bit    |
+      |          |                                   |                                      | shift operation, except the vacated    |
+      |          |                                   |                                      | bits are filled from the overflow,     |
+      |          |                                   |                                      | rather than being set to zero.  The    |
+      |          |                                   |                                      | width of the output is set equal to    |
+      |          |                                   |                                      | the width of the input.                |
+      |          |                                   |                                      |                                        |
+      |          |                                   |                                      | ``rotl(a, n) == rotr(a, -n)``.         |
       +----------+-----------------------------------+--------------------------------------+----------------------------------------+
-      | rotr     | ``bit[n]``, (``int`` or ``uint``) | ``bit[n]``                           |                                        |
+      | rotr     | ``bit[n]``, ``int``               | ``bit[n]``                           | Rotate the bits in the representation  |
```

Comment:
Same question as I asked for `rotl`.

Reply:
Updated to uint and bit only in https://github.com/openqasm/openqasm/pull/366/commits/307fa094e8be792a845623101323999a452ab261